src/components/DropdownButton.vue

```diff
@@ -11,7 +11,7 @@
       <button
         ref="dropdownButton"
         type="button"
-        class="dropdown-button ml-1 rounded-r-sm rounded-l-none"
+        class="dropdown-button ml-1 rounded-r-sm rounded-l-none w-14"
```

Comment:
I don't love the idea of a hardcoded width inside of this component, I'd rather see this be `w-full` and do the `w-14` on a parent at the page, implementation-level.

Reply:
I think a fixed width for the dropdown arrow button is fine... the download button should expand to fill the remaining space (`w-full` I guess?). I tried to implement that with flex but I guess I didn't do it right? I don't think we need to actually set explicit widths on anything? Except maybe a min-width on the download button in case the word "download" is very short in any given language.

Reply:
Oh i totally lazily missed that this was on the dropdown arrow button, oops. I agree with your thoughts @sarayourfriend